Video-Transformers/README.md

```diff
@@ -0,0 +1,151 @@
+
+* Set the `type` parameter to `OTC_MEDIA_TRANSFORMER_TYPE_CUSTOM` (defined in the SDK).
+  This indicates that you are creating a custom transformer.
+* Set the `name` parameter to a unique name for your transformer.
+* Set the `properties` parameter `NULL`. (This parameter is used when using a transformer from
+  the Vonage Media Library.)
+* Set the `callback` parameter to a callback function. This function is an instance of
+  the `video_transform_callback` type, defined in the SDK. This function has two parameters:
+  `user_data` -- see the next parameter -- and `frame` -- an instance of type
+  of type `otc_video_frame` (defined in the SDK) passed into the callback function when
+  there is video frame data available. Transform the video frame data in the callback function.
+* Set the `userData` parameter (optional) to user data to be passed in the callback function.
+
+Here is a basic example:
+
+```c
+void on_transform_logo(void* user_data, struct otc_video_frame* frame)
+{
+    // implement transformer on the otc_video_frame data
+}
+
+otc_video_transformer *logo = otc_video_transformer_create(
+  OTC_MEDIA_TRANSFORMER_TYPE_CUSTOM,
+  "blacknwhite",
+  NULL,
+  on_transform_logo,
+  NULL
+);
+```
+
+After you create the transformer, you can apply it to a publisher using the
+<a href="developer/sdks/mac/reference/publisher_8h.html#a06dec21cf056dbbe1ddadd83c8e3488f"><code>otc_publisher_set_video_transformers()</code></a>
+function:
+
+```c
+// Array of video transformers
+otc_video_transformer *video_transformers[] = {
+  logo
+};
+
+otc_publisher_set_video_transformers(publisher, video_transformers, 1);
+```
+
+## Creating a custom audio transformer
+
+Use the <a href="/developer/sdks/mac/reference/media__transformer_8h.html#a15c109155502af31df8f61be5174f202"><code>otc_audio_transformer_create()</code></a>
+function to create an audio transformer.
+
+* Set the `type` parameter to `OTC_MEDIA_TRANSFORMER_TYPE_CUSTOM` (defined in the SDK).
+  This indicates that you are creating a custom transformer. (In this beta version, no
+  predefined audio transformers from the Vonage Media Library are supported.)
+* Set the `name` parameter to a unique name for your transformer.
+* Set the `properties` parameter `NULL`. (This parameter is used when using a transformer from
+  the Vonage Media Library.)
+* Set the `callback` parameter to a callback function. This function is an instance of
+  the `audio_transform_callback` type, defined in the SDK. This function has two parameters:
+  `user_data` -- see the next parameter -- and `frame` -- an instance of type
+  of type `otc_audio_data` (defined in the SDK) passed into the callback function when
+  there is audio data available. Transform the audio data in the callback function.
+* Set the `userData` parameter (optional) to user data to be passed in the callback function.
+
+Here is a basic example:
+
+```c
+void on_transform_audio(void* user_data, struct otc_video_frame* frame)
```

Comment:
struct otc_video_frame* should be changed to otc_audio_data?

Reply:
@goncalocostamendes can you please check this.

Reply:
Well caught! Actually since this sample is only targetting video transformers, I will remove the entire audio transformer info